daemonset.go

```diff
@@ -27,7 +27,7 @@ func updateContainersDaemonSet(daemonSet *v1beta1.DaemonSet, envVars []v1.EnvVar
 	containers := []v1.Container{}
 
 	for _, c := range podSpec.Containers {
-		c.Env = envVars
+		c.Env = append(c.Env, envVars...)
```

Comment:
That's a good point. I added a merge function to favor vars specified in files over those in the resource doc.